config/src/test/java/org/springframework/security/config/annotation/method/configuration/DelegatingReactiveMessageService.java

```diff
@@ -47,7 +49,7 @@ public String notPublisherPreAuthorizeFindById(long id) {
 	}
 
 	@Override
-	@PostAuthorize("returnObject?.contains(authentication?.name)")
```

Comment:
Changing the tests means we are potentially breaking backwards compatibility. We need to leave tests alone and ensure that any changes that we make don't break the existing tests.